src/zzz_od/application/hollow_zero/lost_void/lost_void_app.py

```
@@ -121,9 +128,44 @@ def choose_lost_void(self) -> OperationRoundResult:
+        # 悬赏委托和基础次数都完成了
+        if self._points_reward_finished and self.ctx.lost_void_record.is_finished_by_day():
+            return self.round_success("已完成悬赏委托")
+
         mission_name = self.ctx.lost_void_config.mission_name
         return self.round_by_goto_screen(screen_name=f'迷失之地-{mission_name}')
 
+    # 检查悬赏委托直到成功完成检测, 并保存结果
+    def _check_and_set_points_reward(self) -> None:
+        if not self._need_check_points_reward:
+            return
+        for i in range(30):
```

Ruff (0.13.3)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)